src/main/java/com/twilio/taskrouter/Workflow.java

```diff
@@ -1,19 +1,15 @@
 package com.twilio.taskrouter;
 
-import com.fasterxml.jackson.annotation.JsonAutoDetect;
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.*;
```

Comment:
I would prefer to import individual classes instead of importing all of them. What do you think?